README.md

```diff
@@ -1,10 +1,112 @@
 
 This is a fork of the official Akka's JDBC plugin, aimed to support PostgreSQL 11 features such as partitions, arrays, BRIN indexes and others. 
+It is a fork of the official Akka's JDBC plugin, aimed to support PostgreSQL 11 features such as partitions, arrays, BRIN indexes and others.
+All of this to reduce RAM usage and index size by the database.  
 
 ## Documentation
+## How to enable Akka Persistence Postgres in your project
 
 * [Akka Persistence JDBC 3.5.x documentation](https://doc.akka.io/docs/akka-persistence-jdbc/3.5/)
 * [Snapshot documentation](https://doc.akka.io/docs/akka-persistence-jdbc/snapshot/)
+To use this plugin prior default one, add the following to application.conf:
+```hocon
+akka.persistence {
+  journal.plugin = "postgres-journal"
+  snapshot-store.plugin = "postgres-snapshot-store"
+}
+```
+and for persistence query
+```scala
+PersistenceQuery(system).readJournalFor[JdbcReadJournal](JdbcReadJournal.Identifier)
+```
+
+## Difference in a compare to Akka Persistence JDBC plugin
+
+### BRIN index on ordering
+Akka-persistence-jdbc uses a B-Tree index on the ordering column. It has a few advantages in comparison to BRIN:  
+it has faster data access, it can guarantee uniqueness of values. However, it's size grow very fast and consume more and more RAM.
+BRIN index does not have information about each record in the table, it knows which value range are persisted in which row range.
+Thanks to that it has a smaller memory footprint. 
+More information about [BRIN index](https://www.postgresql.org/docs/11/brin-intro.html) you can read in official Postgres documentation.   
+
+You can also apply the BRIN index on the ordering column for your akka-persistence-jdbc setup. It does not need any changes in code.
+
+### Tags as an array of int
+Akka-persistence-jdbc store all tags in a single column as String separated by commas. It causes that plugin searching for events by tag uses `tags like '%tag_name%'` statement. 
+It isn't perfect as it can return events which have similar tag names as a searched tag, eg: 
+query looking for events with 'car' tag return also events with 'blue-car' tag. From version 4.0.0 extra events are filtered out in Scala. 
+In future releases, developers of Akka Persistence JDBC is planning to migrate tags to separate table, 
+but it is an extra cost in terms of index size and RAM usage, which is against our goals.
+
+In akka-persistence-postgres we decided to store tag name in separate dictionary table, 
+whereas in the journal table we are keeping the array of ids of tags. We introduce 2 optimizations:
+* [GIN index](https://www.postgresql.org/docs/11/gin.html) on tags column, which makes searching events by tag faster;
+* cache of mapping tag id to tag name to reduce the number of connections to the database.
+The cache is tunable under `TODO` properties.
+
+## Versions of the plugin
+
+Akka Persistence Postgres is available in 2 versions:
```

Comment:
I'd rather say that _Akka Persistence Postgres supports two variants of journal schemas:_

Reply:
ok